server/src/main/java/io/deephaven/server/arrow/ArrowFlightUtil.java

```diff
@@ -551,6 +570,19 @@ public void handleMessage(BarrageProtoUtil.MessageInfo message) {
             public void close() {
                 // no work to do for DoGetRequest close
             }
+
+            private void flushMetrics(long bytesWritten, long writeTm) {
+                if (tableKey == null) {
+                    // metrics for this request are not to be reported
+                    return;
+                }
+
+                try {
+                    BarragePerformanceLog.getInstance().getStaticLogger()
+                            .log(tableId, tableKey, requestTm, queueTm, snapshotTm, writeTm, bytesWritten);
```

Comment:
You're reporting nanos, but your columns claim to be millis. Same for the other `flushMetrics` in `FlightServiceGrpcImpl`.

Reply:
Per our discussion; nanos into the log method which converts to millis.